src/components/Sidebar.jsx

```diff
@@ -1,124 +1,87 @@
-        transition-duration: 0.3s;
+    .font-barlow {
+        font-family: 'Barlow', sans-serif;
     }
-    .fst-btn {
-        background-color: #AAD390;
-        color: #FFFFFF;
-        font-size: 24px;
-        font-weight: 700;
+
+    .font-century-gothic {
+        font-family: 'Century Gothic', sans-serif;
     }
-    .snd-btn {
-        background-color: #FFFFFF;
-        color: #000000;
-        font-size: 20px;
-        font-weight: 500;
+
+    .bg-pink {
+        background-color: #F59489;
     }
-    .sidebar-titles {
-        align-items: start;
-        margin-top: 15px;
-        margin-left: 20px;
-        color: #FFFFFF;
-        opacity: 0.8;
-        font-family: 'Century Gothic', sans-serif;
-        font-size: 22px;
-        font-weight: 700;
+
+    .exit-image {
+        max-width: 12px;
+        max-height: 12px;
     }
-    .sidebar-list-items {
-        text-decoration: none;
-        margin-left: 40px;
-        color: #FFFFFF;
-        font-family: 'Barlow', sans-serif;
-        font-weight: 500;
+
+    .profile-image {
+        width: 110px;
+        height: 110px;
+        border: 8px solid #AAD390;
+        box-shadow: 0px 2px 5px rgba(0, 0, 0, 0.4);
     }
-    .left-arrow {
-        width: 32px;
-        height: 32px;
-        bottom: 10px;
-        left: 40%;
-        position: fixed;
+
+    .sidebar-wrapper h1{
+        opacity: 0.8;
     }
-    .font-barlow {
-        font-family: 'Barlow', sans-serif;
+
+    .sidebar-wrapper .btn:active {
+        background-color: #F59489;
     }
-    .font-century-gothic {
-        font-family: 'Century Gothic', sans-serif;
+
+    .sidebar-wrapper .btn:hover, 
+    .sidebar-wrapper a:hover, 
+    .sidebar-wrapper .btn:active, 
+    .sidebar-wrapper a:active {
+        background-color: #FFBCB5;
     }
 `;
 
 function Sidebar(props) {
     return (
-        <div className="sidebar-wrapper d-flex flex-column">
-            <div className="d-flex flex-column align-items-center">
-                <img
-                    className="perfil-img my-4"
-                    src={PerfilImg}
-                    alt="Perfil"
-                    style={{
-                        width: '100px',
-                        height: '100px',
-                    }}
-                />
-                <button className="btn fst-btn font-century-gothic my-1" type="button">
-                    Novo
-                </button>
-                <button className="btn snd-btn d-flex flex-row my-2">
-                    <img
-                        src={BuscarImg}
-                        alt="Search Icon"
-                        style={{
-                            width: '35px',
-                            paddingRight: '7px',
-                            paddingTop: '3px',
-                        }}
-                    />
-                    <p className="font-barlow">Buscar</p>
+        <div className="sidebar-wrapper d-flex flex-column flex-grow-1">
```

Comment:
```suggestion
        <div className="sidebar-wrapper d-flex flex-column flex-grow-1 bg-coral-red">
```
- Convém a SideBar definir a própria cor de fundo;